src/Core/Domain/Carrier/QueryResult/EditableCarrier.php

```diff
@@ -88,4 +93,34 @@ public function getLogoPath(): ?string
     {
         return $this->logoPath;
     }
+
+    public function getDelay(): array
```

Comment:
```suggestion
    public function getLocalizedDelay(): array
```
Optional, can be done in another PR

Reply:
It's ok in mine too :D